packages/core/src/interpreter.ts

```diff
@@ -651,26 +651,16 @@ export class Interpreter<
             return;
           }
           ref._parent = this; // TODO: fix
-          // If the actor will be stopped right after it's started
-          // (such as in transient states) don't bother starting the actor.
-          if (
-            state.actions.find((otherAction) => {
-              return (
-                otherAction.type === actionTypes.stop &&
-                (otherAction as StopActionObject).params.actor === id
-              );
-            })
```

Comment:
This was the main source of the problem because we had actions looking like this:
```
actions: [stop('a'), invoke('a')]
```

Stop action for `'a'` was simply found in the array and thus the actor was never reinstantiated - despite of the fact that the invoke was coming **after** it.